libs/community/langchain_community/graph_vectorstores/base.py

```diff
@@ -849,14 +850,20 @@ class GraphVectorStoreRetriever(VectorStoreRetriever):
         "mmr_traversal",
     )
 
+    @property
+    def graph_vectorstore(self) -> GraphVectorStore:
+        return cast(GraphVectorStore, self.vectorstore)
+
```

Comment:
I had to add this cast property to get the 3.13 lint to pass. If there is a better way to do this, please let me know.